### PR TITLE
Booking step 1a

### LINKED ISSUE #21: Booking step 1a- show available seats
Show available seats.

As a user I want to see available seats for a movie at a certain time and solon.

Dependencies:
- Booking - initial page

Acceptance Criteria;
- show all seats in the salon
- show available seats

Design:

https://www.figma.com/file/vhNtZfxUiEfTYAYr6yaExM/VadSomHelst?node-id=0%3A1&t=fOSZAsey82JfmXBA-0

### PR DESCRIPTION
**Pull request by jakobpalmborg**
Added javascript rendered html and scss for showing seats. This is static for now since we do not have a backend for the available seats yet.

image

### CODE REVIEW

File: booking.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <title>Booking</title>
     <link rel="stylesheet" href="static/css/main.css" />
     <script type="module" src="src/JS/booking.js" defer></script>
+    <script type="module" src="src/JS/booking.js" defer></script>
```
Comment (mkor92) [inline booking.html:9]: Should change one of the script tags to link "main.js". Right now the hamburger menu doesnt work in booking.html

**Review by mkor92 — APPROVED** [review]
Nice work Jakob! I added a comment about one small fix that you should do before merging this branch.

File: booking.html (hunk 1 above)
Comment (jakobpalmborg) [reply booking.html:9]: Done!

**Review by matodorovic — APPROVED** [review]
Great job Jakob! Looks nice. Another step in booking done. ✅

**Pull request merged**